detox/android/detox/src/main/java/com/wix/detox/reactnative/idlingresources/SerialExecutorReflected.kt

```diff
@@ -0,0 +1,16 @@
+package com.wix.detox.reactnative.idlingresources
+
+import org.joor.Reflect
+import java.util.concurrent.Executor
+
+class SerialExecutorReflected(executor: Any) {
```

Comment:
See [here](https://github.com/react-native-community/async-storage/blob/9823e891f32d2c7d13098401a37e092ec025d07f/android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java#L50)

As they also point out, originally, this is an Android class (see [here](https://github.com/aosp-mirror/platform_frameworks_base/blob/master/core/java/android/os/AsyncTask.java#L297))